src/ClientHandle.cpp

```diff
@@ -77,11 +77,11 @@ cClientHandle::cClientHandle(const AString & a_IPString, int a_ViewDistance) :
 	m_BlockDigAnimStage(-1),
 	m_BlockDigAnimSpeed(0),
 	m_BlockDigAnimX(0),
-	m_BlockDigAnimY(256),  // Invalid Y, so that the coords don't get picked up
+	m_BlockDigAnimY(cChunkDef::Height + 1),  // Invalid Y, so that the coords don't get picked up
```

Comment:
the exact value of cChunkDef::Height is 256. So yes im checking for 257. I did it so it's really clear, that this is an invalid value, which was the purpose of the "256" in the original version. Just cChunkDef::Height would have been enough.